### ISSUE TITLE
Notification Android setProgress didn't work

### ISSUE THREAD

**Issue by NicolasHanzel** [invertase/react-native-firebase#1167]
### Issue

According to your example in your documentation, I call setProgress but the notification is not displayed. 

Example for progress bar in notification here : https://developer.android.com/training/notify-user/build-notification#progressbar

Do you have an idea?

```
const notification = new firebase.notifications.Notification()
.setNotificationId('test')
.setTitle('Progress')
.setBody('Hello World')
.setSound('default')
.android.setChannelId('testchannel')
.android.setCategory('progress')
.android.setProgress(100, 0, false)     

firebase.notifications().displayNotification(notification)`
```

### Environment

1. Application Target Platform: Android

2. Development Operating System:  macOs 10.13.4

3. Build Tools:  0.55.1

4. React Native version: 0.55.1

5. RNFirebase Version: 4.2.0

6. Firebase Module: Notifications

**Comment by NicolasHanzel** [comment]
I found the issue, 

**Replace** line 203 of android/src/main/java/io/invertase/firebase/notifications/DisplayNotificationTask.java

`Bundle progress = android.getBundle("lights"); `

**By** 

`Bundle progress = android.getBundle("progress");`

**Comment by chrisbianca** [comment]
@NicolasHanzel thanks for highlighting this, I've just pushed up a fix that will make it into the next release

**Comment by Salakar** [comment]
@NicolasHanzel thanks for this, fix is now live in the [v4.3.0 release](https://github.com/invertase/react-native-firebase/releases/tag/v4.3.0).

---
Loving `react-native-firebase` and the support we provide? Please consider supporting us with any of the below:

  - 👉  Back financially via [Open Collective](https://opencollective.com/react-native-firebase/donate)  
  - 👉  Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter 
  - 👉  Star this repo on GitHub ⭐️
